float64 time
int32 nr_batch
int32 nr_persons

sensor_msgs/Image[] detected_persons
int32[] x_positions